pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -394,6 +409,22 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     return {"class": "reduce", "mode": self.mode, "axes": self.axes, "from": self._get_input_layer_name(input)}
 
 
+class Length(Module):
+  """
+  Wraps RETURNN LengthLayer.
+  """
+  is_original_torch_module = False
+
+  def __init__(self, axis: int):
+    super(Length, self).__init__()
+    self.axis = axis
+
+  def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
+    return {
+      "class": "length", "axis": self._get_input_axis_to_returnn(input, self.axis),
+      "from": self._get_input_layer_name(input)}
```

Comment:
We need to somehow specify `int32` for the returned tensor dtype.

Reply:
Hm or maybe not.

Reply:
No, actually, I think the tensor in the end will be created via:
```
res = module.make_output_tensor_from_returnn(inputs_flat=inputs_flat, layer=layer)
```
And that will do:
```
tensor = Tensor(*torch_shape, numpy_array=numpy_array, dtype=layer.output.dtype)
```

Reply:
On RETURNN side or torch side?